File: JavaScript/areal.js
/*
 *	Problem: "A Real Challenge"
 */
 
const readline = require('readline');

const rl = readline.createInterface({
 input: process.stdin,
 output: process.stdout
});

rl.on('line', (line) => {
 var area = parseInt(line);
 var perimeter = (Math.sqrt(area) * 4);
 console.log(perimeter);	 
});

File: C++/areal.cpp
/*
 *	Problem: "A Real Challenge"
 */
 
#include <iostream>
#include <iomanip>
#include <cmath>

using namespace std;

/*-----main-----*/
int main(void) {
	long int area;
	cin >> area;
	
	double perimeter = (sqrt(area) * 4);
	cout << setprecision(7) << perimeter << endl;

	return 0;
}
